add description of odd characters (#27)
IBM/MAX-Text-Summarizer: README.md

}
```

> As you can see, the text summarizer not only gives the summary text but also filters out some special characters such as `-lrb- `, `-rrb-` and `@highlight`, etc, from the [Daily Mail](https://github.com/abisee/cnn-dailymail) dataset. 

Reviewer: ```suggestion
In this example, the text summarizer not only gives the summary text but also filters out some special characters such as `-lrb-` (representing `(`), `-rrb-` (representing `)`), `@highlight`, etc, from the [Daily Mail](https://github.com/abisee/cnn-dailymail) dataset.
```

Author: Thanks @xuhdev. I'd use `comparing to...` to show the difference.

> Comparing to the original [sample email](samples/sample1.json), the text summarizer not only gives the summary text but also filters out some special characters such as `-lrb-` (representing `(`), `-rrb-` (representing `)`), etc, from the [Daily Mail](https://github.com/abisee/cnn-dailymail) dataset. 

Reviewer: ```suggestion
The text summarizer preserves in the output summary text some special characters such as `-lrb-` (representing `(`), `-rrb-` (representing `)`), etc. that appear in the input [sample](samples/sample1.json), which is excerpted from the [Daily Mail](https://github.com/abisee/cnn-dailymail) dataset.
```

Author: Thanks @xuhdev.  Hi @MLnick, may I also have your suggestions? Thanks.

Reviewer 2: This seems fine
